file-formats/sfpf/examples/z_aff_example_sfpf.sfpf.json

```diff
@@ -0,0 +1,15 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example SFPF for ABAP file formats",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "cloudDevelopment"
+  },
+  "props": {
+    "name": "TEMPLATE_NAME",
```

Comment:
Sorry, same as for the `description` before. Is this the object name? If so, we should remove it. The object name should be the file name.